app/controllers/api/sf311_cases_controller.rb

```diff
@@ -1,9 +1,27 @@
 class Api::Sf311CasesController < ApplicationController
   def index
-    @lane_blockages = Sf311Case.includes(:sf311_case_metadatum)
-    @lane_blockages = @lane_blockages.where('requested_datetime >= ?', params[:start_time]) if params[:start_time]
-    @lane_blockages = @lane_blockages.where('requested_datetime <= ?', params[:end_time]) if params[:end_time]
-    @lane_blockages = @lane_blockages.paginate(page: (params[:page] || 1), per_page: (params[:per_page] || 30))
+    @lane_blockages =
+      Sf311Case
+        .left_outer_joins(:sf311_case_metadatum)
+        .select('sf311_cases.*,sf311_case_metadata.*')
+
+    if params[:start_time]
```

Comment:
Seems like you're not a fan of the ruby one-liners!

Reply:
I'm not. :) I could give reasons if you care but I think this is easier to read.